ansible/roles/postgresql/tasks/main.yml

```diff
@@ -98,11 +99,31 @@
     name: "{{ item.name }}"
     state: present
     port: "{{ postgresql_port }}"
-    owner: "{{ localsettings.PG_DATABASE_USER }}"
+    owner: "{{ item.owner }}"
   with_items: postgresql_dbs
   when:
     pg_install == True
 
+- name: Grant PostgreSQL privileges
+  sudo_user: postgres
+  postgresql_privs: type=database database="{{ item.1.db_name }}" roles="{{ item.0.username}}" privs="{{ item.1.priv}}"
+  with_subelements:
+    - "{{ postgres_users.values() }}"
+    - privileges
+    - flags:
+      skip_missing: True
```

Comment:
We want to grant `CONNECT` privilege to readonly users as well, so this should be here, no?